examples/using-sdk/helper/env/main.go

```diff
@@ -0,0 +1,37 @@
+/*
+ * .-'_.---._'-.
+ * ||####|(__)||   Protect your secrets, protect your business.
+ *   \\()|##//       Secure your sensitive data with Aegis.
+ *    \\ |#//                    <aegis.ist>
+ *     .\_/.
+ */
+
+package main
+
+import (
+	"fmt"
+	"github.com/shieldworks/aegis/sdk/sentry"
+	"strings"
+)
+
+func main() {
+	d, err := sentry.Fetch()
+	if err != nil {
+		msg := err.Error()
+
+		if strings.Contains(msg, "Secret does not exist") {
+			fmt.Print("NO_SECRET")
+			return
+		}
+
+		fmt.Print("ERR_SENTRY_FETCH_FAILED")
+		fmt.Print(" ", err.Error())
```

Comment:
This is a different case. We don’t know if the secret is deleted or not because we were not able to fetch the secret.